src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -222,15 +222,15 @@ public final class Constants {
 
     public static final String HAZELCAST_EXERCISE_CACHE = HAZELCAST_QUIZ_PREFIX + "exercise-cache";
 
-    public static final long MONITORING_CACHE_RESET_DELAY = 60 * 30; // 30 minutes in seconds
+    public static final long LIVE_STATISTICS_CACHE_RESET_DELAY = 60 * 30; // 30 minutes in seconds
```

Comment:
It's not clear these constants belong to exam live statistics. Maybe rename them with “exam” or add a comment.

Reply:
This will be removed when the database pr is merged - so we can ignore it here :D